dashboard/views/users.ejs

```diff
@@ -52,13 +52,16 @@
                           </div>
                         </div>
 			<script>
-			window.addEventListener('localized', function() {  // HACK: Force localization of role combo
-				var selection = document.getElementsByClassName("select2-selection__rendered");
-				if (selection && selection[0] && typeof selection[0].innerHTML == "string" &&  document.getElementById("user-type-select2")) {
+                window.addEventListener('localized', function() {  // HACK: Force localization of role combo
```

Comment:
Warning: don't mix indentation style in the file

Reply:
Fixed it in my latest commit.